### PR TITLE
fix-vimWebSessionPaloAltoCEF

### PR DESCRIPTION
**Pull request by KustoKing**
Required items, please complete
   
   Change(s):
   - Fix ASimMatchingIpAddr

   Reason for Change(s):
   - ASimMatchingIpAddr returned wrong value

   Testing Completed:
   - Testing from Fortinet parser

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Review by vakohl — CHANGES_REQUESTED** [review]

File: Parsers/ASimWebSession/Parsers/vimWebSessionPaloAltoCEF.yaml [hunk 1]
```diff
@@ -106,11 +108,13 @@ ParserQuery: |
       | where (array_length(url_has_any) == 0 or RequestURL has_any (remove_protocol_from_list(url_has_any)))
       | where (array_length(httpuseragent_has_any) == 0 or RequestClientApplication has_any (httpuseragent_has_any))
```
[the remark this answers is not in the document]
Comment (KustoKing) [reply Parsers/ASimWebSession/Parsers/vimWebSessionPaloAltoCEF.yaml:109]: I haven't seen any actual UserAgent data in the RequestClientApplication, so I did not map it. I did switch the Product and the vendor.

[the remark this answers is not in the document]
Comment (vakohl) [reply Parsers/ASimWebSession/Parsers/vimWebSessionPaloAltoCEF.yaml:109]: thanks @KustoKing . I have seen 'RequestClientApplication' values from PANOS for other customers. Can we set and project both HttpUserAgent and UserAgent?

We are making changes to only VIM parser I think, can we apply all the changes you made to the ASIM parser as well?

**Review by vakohl — CHANGES_REQUESTED** [review]
requested changes. details in the previous comment

File: Parsers/ASimWebSession/Parsers/vimWebSessionPaloAltoCEF.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (KustoKing) [reply Parsers/ASimWebSession/Parsers/vimWebSessionPaloAltoCEF.yaml:109]: Sure I added it and updated ASIM

**Comment by KustoKing** [comment]
> requested changes. details in the previous comment

changes should be there

**Review by KustoKing — COMMENTED** [review]
Made changes

**Comment by v-atulyadav** [comment]
Hi @KustoKing, Thank you for the updates. We will provide you with an update as soon as possible. Thanks

**Review by vakohl — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**